packages/react/jest.setup.ts

```diff
@@ -1,8 +1,7 @@
-import { jest } from "@jest/globals";
 import "@testing-library/jest-dom/extend-expect";
 import crossFetch from "cross-fetch";
 
-jest.setTimeout(process.env.CI == "vscode-jest-tests" ? 20 * 60 * 1000 : 5 * 1000);
+//jest.setTimeout(process.env.CI == "vscode-jest-tests" ? 20 * 60 * 1000 : 5 * 1000);
```

Comment:
Why comment out this line? 🤔

Reply:
whoops that was an oversight. Good catch.